build/release/README.md

```diff
@@ -55,26 +83,28 @@ To publish a new patch release build, follow these steps:
 
 1. Make sure all needed PRs are merged to the release branch
 2. Check that integration tests are green (except intermittent issues)
-3. Open a PR to update the version, and merge it
+3. Open a PR to update the doc/manifest image tag versions, and merge it
+   For example: https://github.com/rook/rook/pull/13301
 4. Tag the branch:
 
     ```console
     # make sure no files are checked out locally, then proceed:
-    git checkout <branch> # e.g. release-1.10
-    git pull
+    git fetch --all
+    git checkout <branch> # e.g. release-1.12
+    git reset --hard upstream/release-1.12
     # set to the new release
-    tag_name=<release version> # e.g., v1.10.9
```

Comment:
I was finding it a little hard to read the script when this was lower case, so I all-capped it to make it clear it's a shell var.